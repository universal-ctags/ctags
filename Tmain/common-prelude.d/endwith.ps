(input.md) (.md) _endwith ==
(input.md) (.nd) _endwith ==
(.md) (input.md) _endwith ==
() (input.md) _endwith ==
(input.md) () _endwith ==
() () _endwith ==
(input.md) (input.md) _endwith ==
