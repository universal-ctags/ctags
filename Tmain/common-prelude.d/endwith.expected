true
false
false
false
true
true
true
